integrations/vllm.md

```diff
@@ -3,29 +3,21 @@ layout: integration
 name: vLLM Invocation Layer
```

Comment:
Maybe we can rename this integration to "vLLM" only?
Invocation layers refers to the specific implementation for 1.x